test/util/bitcoin-unit-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+PROCESSES=4
+DIR="$(dirname "$0")"
+
+cat $DIR/../../src/{,wallet/,qt/}test/*.cpp | fgrep BOOST_FIXTURE_TEST_SUITE | cut -d '(' -f 2 | cut -d ',' -f 1 | shuf | xargs -n 1 -P $PROCESSES -I "{}" -- bash "$DIR/bitcoin-unit-test.sh" "{}/*"
```

Comment:
`shuf` is not available under OS X by default. Is `sort -R` sufficiently portable?

Reply:
Fixed.